pkg/report/linux.go

```diff
@@ -661,3 +711,93 @@ var linuxOopses = []*oops{
 		[]*regexp.Regexp{},
 	},
 }
+
+func isCorrupted(desc string, text string) bool {
+	if !strings.Contains(text, "Call Trace") && !strings.Contains(text, "backtrace") {
+		// Text must contain 'Call Trace' or 'backtrace'.
+		return true
+	}
+	corruptedDescRegExps := []*regexp.Regexp{
+		// 'kernel panic: Fatal exception' is usually printed after BUG,
+		// so if we captured it as a report description, that means the
+		// report got truncated and we missed the actual BUG header.
+		regexp.MustCompile(`kernel panic: Fatal exception`),
```

Comment:
If we add corrupted field to oopsFormat (see below), then we can fold this into linuxOopses table.

Reply:
Done in #427.